### PR TITLE
Fix headers and add stubs to QuartzCore

### PR DESCRIPTION
**Pull request by theiostream**
Fix header issues and add stubs to the QuartzCore library.

### CODE REVIEW

File: GNUmakefile [hunk 1]
```diff
@@ -28,7 +28,7 @@ SVN_BASE_URL = svn+ssh://svn.gna.org/svn/gnustep/libs
 
 SUBPROJECTS = Source
 SUBPROJECTS += Tests
-SUBPROJECTS += Demo
+#SUBPROJECTS += Demo
```
Comment (fredkiefer) [inline GNUmakefile:31]: Why did you remove the compilation of Demo?

File: Headers/QuartzCore/CALayer.h [hunk 2]
```diff
@@ -30,34 +30,34 @@
...
+extern NSString *const kCAGravityBottomRight;
+extern NSString *const kCATransition;
 
 @class CAAnimation;
 @class CARenderer;
 
 #import "QuartzCore/CAAction.h"
 
-#if GNUSTEP
+//#if GNUSTEP
```
Comment (fredkiefer) [inline Headers/QuartzCore/CALayer.h:56]: Could you please explain why you had to change this? The old code is a horrible workaround, but using it everywhere looks even worse.

File: Headers/QuartzCore/CALayer.h [hunk 3]
```diff
@@ -30,34 +30,34 @@
 #import "QuartzCore/CABase.h"
 #import "QuartzCore/CAMediaTiming.h"
 #import "QuartzCore/CATransform3D.h"
-#if GNUSTEP
+//#if GNUSTEP
```
Comment (fredkiefer) [inline Headers/QuartzCore/CALayer.h:33]: Either leave the #if in place or remove it completely. In the later case you will have to add some explanation why.
The same is true for all similar changes you made.

File: Headers/QuartzCore/CALayer.h [hunk 4]
```diff
@@ -207,6 +208,10 @@ NSString *const kCATransition;
 - (CGAffineTransform) affineTransform;
 - (void) setAffineTransform: (CGAffineTransform)affineTransform;
 
+@property (nonatomic, assign) CGColorRef borderColor;
+
+@property (nonatomic, assign) CGFloat contentsScale;
```
Comment (fredkiefer) [inline Headers/QuartzCore/CALayer.h:213]: Where do you synthesise these two variables?

**Review by fredkiefer — CHANGES_REQUESTED** [review]
At least some explanation is still needed.

**Comment by ivucica** [comment]
@theiostream @fredkiefer I have opted to merge this without the unclear commit d20b8e7e93a38172cfafabf7e8fe5c5cf67fe590.

File: Headers/QuartzCore/CALayer.h (hunk 4 above)
Comment (ivucica) [reply Headers/QuartzCore/CALayer.h:213]: It is synthesized later. I'll just add a comment that `assign` is correct, too.

`contentsScale` is on the other hand correct, but not synthesized. I'm adding this. Presumably this worked because Clang.

I'd also prefer if this was moved elsewhere, up, with other animatable properties. I'll remedy this.
